tracer/test/Datadog.Trace.Tests/Propagators/DatadogPropagatorTests.cs

```diff
@@ -70,11 +82,13 @@ public void Inject_IHeadersCollection_Propagation_Disabled()
                 {
                     new("x-datadog-trace-id", TraceId.ToString(InvariantCulture)),
                     new("x-datadog-parent-id", SpanId.ToString(InvariantCulture)),
+
+                    // TODO: sampling priority, origin
```

Comment:
Was this TODO supposed to be filled in?

Reply:
Yes, thanks!

Reply:
Done. I left that TODO because I wanted to expand this test to also check for sampling priority and origin headers.